doc/api/worker_threads.md

```diff
@@ -561,6 +561,10 @@ changes:
     occur as described in the [HTML structured clone algorithm][], and an error
     will be thrown if the object cannot be cloned (e.g. because it contains
     `function`s).
+  * `transferList` {Object[]} If you pass one or multiple `MessagePort`
+    like objects in `workerData`, you will need to provide `transferList` for
+    those items. Otherwise, [ERR_MISSING_MESSAGE_PORT_IN_TRANSFER_LIST][] will
+    be thrown.
```

Comment:
```suggestion
  * `transferList` {Object[]} If one or more `MessagePort`-like objects
    are passed in `workerData`, a `transferList` is required for those
    items or [ERR_MISSING_MESSAGE_PORT_IN_TRANSFER_LIST][] will be
    thrown.
```

Avoiding use of `you` in the docs :-)

Reply:
Get it.

So the final result will be:

```
  * `transferList` {Object[]} If one or more `MessagePort`-like objecst are passed in `workerData`, a `transferList` is required for those items or [ERR_MISSING_MESSAGE_PORT_IN_TRANSFER_LIST][] will be thrown.
```

Am I right?

Reply:
May resolved in  78b3e7b